src/main/java/com/revature/RevRelay/security/WebSecurityConfig.java

```diff
@@ -132,31 +134,33 @@ protected void configure(final HttpSecurity http) throws Exception {
      * @return CorsConfigurationSource configured with applyPermitDefaultValues on
      *         all endpoints.
      */
+
+    /*
     @Bean
```

Comment:
This bean needs to be commented out or deleted (preferably commented out), otherwise there are two beans providing CorsConfigurationSource which is a problem.